src/compiler/compile/render-dom/wrappers/Window.ts

```diff
@@ -92,29 +92,29 @@ export default class WindowWrapper extends Wrapper {
 
 				renderer.meta_bindings.add_block(deindent`
 					if (${condition}) {
-						window.scrollTo(${x || 'window.pageXOffset'}, ${y || 'window.pageYOffset'});
+						@_scrollTo(${x || '@_pageXOffset'}, ${y || '@pageYOffset'});
```

Comment:
Typo here, supposed to be `@_pageYOffset`.

Reply:
Actually, both of the offsets need to be `@_window.{offset}` — or else we're binding on module load!

Reply:
Wait, what's the difference between using `pageXOffset` and `window.pageXOffset`?

Reply:
Because we'd generate something like:
```js
import { globals } from 'svelte/internal';
const { pageXOffset } = globals;
```
Binding to the number at module load.

Reply:
Ohh gotcha. Okay.